DSharpPlus.Interactivity/InteractivityExtension.cs

```diff
@@ -556,7 +556,7 @@ public async Task<ReadOnlyCollection<Reaction>> CollectReactionsAsync(DiscordMes
         /// <param name="deletion">Deletion behaviour</param>
         /// <param name="token">A custom cancellation token that can be cancelled at any point.</param>
         public async Task SendPaginatedMessageAsync(
-            DiscordChannel channel, DiscordUser user, IEnumerable<Page> pages, PaginationButtons buttons = null,
```

Comment:
Should probably add in the documentation that setting it null uses the default buttons